administration/src/bp-modules/self-service/CardSelfServiceForm.tsx

```diff
@@ -43,12 +43,27 @@ const CardSelfServiceForm = ({
 }: CardSelfServiceFormProps): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
   const projectConfig = useContext(ProjectConfigContext)
+  const [formSendAttempt, setFormSendAttempt] = useState(false)
+  const [touchedFullName, setTouchedFullName] = useState(false)
   const [openDataPrivacy, setOpenDataPrivacy] = useState<boolean>(false)
   const [openReferenceInformation, setOpenReferenceInformation] = useState<boolean>(false)
   const cardValid = isValid(card, { expirationDateNullable: true })
   const appToaster = useAppToaster()
 
+  const updateFullName = (fullName: string) => {
+    updateCard({ fullName })
+    if (!touchedFullName) {
+      setTouchedFullName(true)
+    }
```

Comment:
:wrench: If this is named like this, I think we should always set `touchedFullName` to `true` on updates.